e2e/cypress/integration/auth_sso/authentication_4_spec.js

```diff
@@ -273,14 +273,25 @@ describe('Authentication', () => {
         // # Open team menu and click on "Invite People"
         cy.uiOpenTeamMenu('Invite People');
 
-        // # Click invite members
-        cy.findByText('Members').click();
+        // # Click invite members if needed
+        cy.get('.InviteAs').then(($inviteAs) => {
+            const hasABToggleTreatment = $inviteAs.find('#inviteAsToggleControl').length > 0;
+            if (hasABToggleTreatment) {
+                if ($inviteAs.find('#inviteMembersLink').length > 0) {
+                    // Has A/B test toggle treatment and is in guest mode.
+                    cy.findByTestId('inviteMembersLink').click();
+                }
+            } else {
+                // Has A/B test radio treatment, so inviteMembersLink is always visible.
+                cy.findByTestId('inviteMembersLink').click();
+            }
+        });
```

Comment:
Thank you for covering both treatments! That makes is easier to update in the future whichever option will be picked up as final feature.